aspnetcore/signalr/messagepackhubprotocol.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Use MessagePack Hub Protocol in SignalR for ASP.NET Core
+author: rachelappel
+description: Add MessagePack Hub Protocol to ASP.NET Core SignalR.
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/04/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/messagepackhubprotocol
+---
+
+# Use MessagePack Hub Protocol in SignalR for ASP.NET Core
+
+By [Brennan Conroy](https://github.com/BrennanConroy)
+
+This article assumes the reader is familiar with the topics covered in [Get Started](xref:signalr/get-started).
+
+## What is MessagePack?
+
+[MessagePack](https://msgpack.org/index.html) is a binary serialization format that is fast and compact. It's useful when performance and bandwidth are a concern because it creates smaller packets compared to [JSON](https://www.json.org/). Because it's a binary format, messages are unreadable when looking at network traces and logs unless the bytes are passed through a MessagePack parser. SignalR has added built-in support for the MessagePack format, and provides APIs for the client and server to use.
```

Comment:
nit: ~~packets~~ **messages**

Reply:
nit: SignalR has ~~added~~ built-in support

(In 3 years when it's not so new, this doc will look a little silly if it says "added")